app/views/templates/base.php

```diff
@@ -125,6 +125,20 @@
   </div>
 
   <script src="/assets/jquery/jquery.min.js?v=<?= VERSION ?>"></script>
+  <script src="/assets/clipboard.js/clipboard.js-built.js?v=<?= VERSION ?>"></script>
+  <script>
+     var clipboard = new Clipboard('.clipboard');
+     clipboard.on('success', function(e) {
+            e.trigger.setAttribute('data-title', 'Translation copied!');
+            e.trigger.setAttribute('class', 'clipboard tooltip');
+            setTimeout(function(arg1) {
+                arg1.trigger.setAttribute('class', 'clipboard')
+            }, 500, e);
+            e.clearSelection();
+        }
+    );
+ </script>
```

Comment:
Can we move this to ```assets/base.js```? The idea would be to not have JavaScript code inside .php files, unless unavoidable (like at the end of the template, where PHP is needed to create the code dynamically).